empress_gui.py

```diff
@@ -429,6 +431,10 @@ def display_tanglegram(self):
         self.tanglegram_window = tk.Toplevel(self.master)
         self.tanglegram_window.geometry("600x600")
         self.tanglegram_window.title("Tanglegram")
+        # Bring the new tkinter window to the front
+        self.tanglegram_window.attributes('-topmost', True)
+        self.tanglegram_window.focus_force()
+        self.tanglegram_window.bind('<FocusIn>', self.OnFocusIn)
```

Comment:
What is this line doing?

Reply:
When the user interacts with this window (i.e. bringing focus to it), this window will not be fixed in the front any more, so this window will only be brought to the front of all windows tentatively when it is opened instead of permanently staying in the front of all windows. This is achieved by calling the self.OnFocusIn function.

Reply:
Rename `self.OnFocusIn` to `self.bring_to_front`.

Reply:
Ok!